scripts/check-type-dependencies.js

```diff
@@ -86,11 +86,8 @@ function checkTypes(pkg) {
     'utf8',
   );
   const deps = (typeDecl.match(/from '.*'/g) || [])
-    .map(match => match.replace(/^.*from '(.*)'.*$/, '$1'))
-    .filter(
-      n =>
-        !(n.startsWith('.') || n === pkg.name || n.startsWith(`${pkg.name}/`)),
-    );
+    .map(match => match.replace(/from '(.*)'/, '$1'))
```

Comment:
this regex is wrong - either use mine, or add back the `g`, right?

Reply:
The global matching is done above, this just extracts the from bit

Reply:
```
> "export * from './service/router';".replace(/from '(.*)'/, '$1')
< "export * ./service/router;"
```

Reply:
```
> "export * from './service/router';".replace(/^.*from '(.*)'.*$/, '$1')
< "./service/router"
```

Reply:
The output of `typeDecl.match(/from '.*'/g)` is what we're processing here

Reply:
haha oh derp, and then we don't map the full string, we map a capture group, kk